index.html

```diff
@@ -636,12 +637,17 @@ <h3>
           global object</a> for <a>this</a>. See [=/HTTP fetch=] for more info.
         </p>
         <p data-dfn-for="PerformanceResourceTiming">
-          The <dfn>responseStart</dfn> getter steps are to <a>convert fetch
+          The <dfn>finalResponseHeadersStart</dfn> getter steps are to <a>convert fetch
```

Comment:
I *thought* we settled on `finalResponseHeaderStart` (singular rather than plural). Did I get it wrong?

Reply:
Nope. Look at #345. It's when the headers (plural) arrive.

Reply:
I don't remember being clear one way or another. I documented `finalResponseHeadersStart` but without too much thought to be honest. I don't feel strongly one way or the other.

Is there any precedent either way?

Reply:
It's "header" in the HTTP spec but [`Headers`](https://fetch.spec.whatwg.org/#headers-class) in `fetch`, especially when web-exposed (`fetch(url).headers`). I think we're closer to `fetch` here and should align with the web-expose notion of `Headers`.

Reply:
I'd prefer headers

Reply:
As @noamr pointed out elsewhere, the [http terminology](https://www.rfc-editor.org/rfc/rfc9110.html#name-header-fields) is (coloquially) "headers" and Fetch has a [Headers](https://fetch.spec.whatwg.org/#headers-class) class, so let's align with that